### PR TITLE
Storybook: Add BaseControl component

### PR DESCRIPTION
**Pull request by fernandovbs**
## Description
Adds baseControl component to storybook

## How has this been tested?
Run ```npm run storybook:dev``` and confirm component shows
Adjust knobs

## Types of changes
Storybook.

### CODE REVIEW

File: packages/components/src/base-control/stories/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * External dependencies
+ */
+import { boolean, text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import BaseControl from '../';
+
+export default { title: 'Components|Base-control', component: BaseControl };
```
Comment (ItsJonQ) [inline packages/components/src/base-control/stories/index.js]: @fernandovbs  Thank you for working on this one. I tested it locally, and it's working as expected. The only thing I'd adjust is the title` Base-control` -> `Base Control`

Thank you!~ 🙏

Comment (fernandovbs) [reply packages/components/src/base-control/stories/index.js]: Hi @ItsJonQ, Got that. 
Thank you for your reply. I'll change it now :smile:

File: packages/components/src/base-control/stories/index.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { boolean, text } from '@storybook/addon-knobs';
  */
 import BaseControl from '../';
 
-export default { title: 'Components|Base-control', component: BaseControl };
+export default { title: 'Components|Base Control', component: BaseControl };
```
Comment (ItsJonQ) [inline packages/components/src/base-control/stories/index.js]: @fernandovbs Awesome! One final tiny nitpick ⛏ (apologies!). I think it should be `BaseControl` (no space). Just like the React component name :).

Thank you!

Comment (fernandovbs) [reply packages/components/src/base-control/stories/index.js]: No problem :smile:. Done!

**Review by ItsJonQ — APPROVED** [review]
🚀 from me! Thank you @fernandovbs !

**Pull request merged**